students/meeryu/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from users.views import UserView
+
+urlpatterns=[
+    path('/userlist', UserView.as_view())
```

Comment:
- 현재 작성하신대로라면 엔드포인트 주소가 `:8000/users/userlist` 입니다. 해당 api는 회원가입이지 유저 목록을 반환하는 api가 아닙니다. 모든 부분에서 네이밍을 올바르게 해주세요!
- 추가로 View도 `UserView`가 아니라 보다 명확하게 작성해주시면 좋을 것 같습니다.
  - `UserView`로 작성했을때 로그인 View는 어떻게 처리해야할지 고민해보시면 아마 답이 나올 것 같아요.
```suggestion
    path('/signup', SignUpView.as_view())
```